app/src/main/java/com/chess/clock/adapters/TimeControlAdapter.java

```diff
@@ -6,17 +6,17 @@
 import android.widget.ArrayAdapter;
 
 import com.chess.clock.R;
-import com.chess.clock.engine.TimeControl;
+import com.chess.clock.engine.TimeControlWrapper;
 import com.chess.clock.views.TimeControlCheckedTextView;
 
 import java.util.ArrayList;
 
 
-public class TimeControlAdapter extends ArrayAdapter<TimeControl> {
+public class TimeControlAdapter extends ArrayAdapter<TimeControlWrapper> {
```

Comment:
I'm intrigued by this new wrapper class...I will look for it later on in this PR